### PR TITLE
Test against Django 1.11

### PR DESCRIPTION
**Pull request by edmorley**
The `>=1.11a1,<2.0` specifier will avoid having to adjust the version as the Django 1.11 pre-release cycle continues.

Also stops testing Django master against Python 2, since they dropped support for it.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,11 +11,12 @@ env:
    - DJANGO="Django>=1.8,<1.9"
    - DJANGO="Django>=1.9,<1.10"
    - DJANGO="Django>=1.10,<1.11"
+   - DJANGO="Django>=1.11a1,<2.0"
```
Comment (auvipy) [inline .travis.yml]: 1.11 is proper now

Comment (edmorley) [inline .travis.yml]: Updated.

**Comment by edmorley** [comment]
@joke2k please could you merge this?

**Comment by Surgo** [comment]
👍

**Pull request merged**